src/catchup/DownloadApplyTxsWork.cpp

```diff
@@ -129,4 +131,20 @@ DownloadApplyTxsWork::onSuccess()
 {
     mLastApplied = mApp.getLedgerManager().getLastClosedLedgerHeader();
 }
+
+std::string
+DownloadApplyTxsWork::getStatus() const
+{
+    auto& hm = mApp.getHistoryManager();
+    auto first = hm.checkpointContainingLedger(mRange.mFirst);
+    auto last = hm.checkpointContainingLedger(mRange.mLast);
+
+    auto checkpointsApplied =
+        (mCheckpointToQueue - first) / hm.getCheckpointFrequency();
+    auto totalCheckpoints = (last - first) / hm.getCheckpointFrequency() + 1;
+    return fmt::format("{}: num checkpoints applied:{}, total checkpoints "
+                       "queued for application:{} ({}%)",
```

Comment:
I would not use `queued` here as it's confusing, in fact maybe just having the number of checkpoints to apply *left* (instead of applied & total) and the percentage is better (gives right enough information).

Reply:
Done